tests/load/pipeline/test_redshift.py

```diff
@@ -41,3 +45,71 @@ def my_source() -> Any:
         assert pip_ex.value.__cause__.data_type == "time"
     else:
         assert pip_ex.value.__cause__.data_type in ("time", "binary")
+
+
+AWS_BUCKET = dlt.config.get("tests.bucket_url_s3", str)
+redshift_with_staging_configs = [
+    DestinationTestConfiguration(
+        destination_type="redshift",
+        staging="filesystem",
+        file_format="jsonl",
+        bucket_url=AWS_BUCKET,
+        extra_info="credential-forwarding",
+    ),
+    DestinationTestConfiguration(
```

Comment:
All of the configurations except this one are already tested. I think you can just add this to staging destinations in `def destinations_configs(` and this will be picked up by many test cases

Reply:
done